PartyPlaylists/PartyPlaylists/Services/RoomSongDataStore.cs

```diff
@@ -0,0 +1,61 @@
+﻿using PartyPlaylists.Contexts;
+using PartyPlaylists.Models.DataModels;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore.Query;
+using Microsoft.EntityFrameworkCore;
+using System.Net.Http;
+
+namespace PartyPlaylists.Services
+{
+    public class RoomSongDataStore : IDataStore<RoomSong>
+    {
+        private readonly HttpClient client;
+        private readonly PlaylistContext playlistContext;
```

Comment:
Rename context with underscores

```suggestion
        private readonly PlaylistContext _playlistContext;
```